tests/drivers/dma/chan_blen_transfer/boards/adp_xc7k_ae350.conf

```diff
@@ -0,0 +1 @@
+CONFIG_ICACHE=n
```

Comment:
Why disable ICACHE for this test?

Reply:
Oh, it's my negligence.
We should disable DCACHE not ICACHE
I have update the source code.

Thank you.